en/getting_started/README.md

```diff
@@ -0,0 +1,96 @@
+  > **Note** Putting this line into a file called **user_config.pri** in the top-level directory (same directory as **qgroundcontrol.pro**) will set this flag on all builds without interfering with the GIT history.
+* **Parallel builds:** For non Windows builds, you can use the `-j#` option to run parellel builds.
+* **Location of built files:** Individual build file results can be found in the `build_debug` or `build_release` directories. The built executable can be found in the `debug` or `release` directory.
+* **If you get this error when running _QGroundControl_**: `/usr/lib/x86_64-linux-gnu/libstdc++.so.6: version 'GLIBCXX_3.4.20' not found.`, you need to either update to the latest *gcc*, or install the latest *libstdc++.6* using: `sudo apt-get install libstdc++6`.
+* **Unit tests:** To run the [unit tests](../contribute/unit_tests.md), build in `debug` mode with `UNITTEST_BUILD` definition, and then copy `deploy/qgroundcontrol-start.sh` script into the `debug` directory before running the tests.
+
+## Optional/OS-Specific Functionality
+
+*QGroundControl* has functionality that is dependent on the operating system and libraries installed by the user. The following sections describe these features, their dependencies, and how to disable/alter them during the build process. These features can be forcibly enabled/disabled by specifying additional values to qmake. 
+
+### Opal-RT's RT-LAB Simulator
+
+Integration with Opal-RT's RT-LAB simulator can be enabled on Windows by installing RT-LAB 7.2.4. This allows vehicles to be simulated in RT-LAB and communicate directly with QGC on the same computer as if the UAS was actually deployed. This support is enabled by default once the requisite RT-LAB software is installed. Disabling this can be done by adding `DEFINES+=DISABLE_RTLAB` to qmake.
+
+### XBee Support
+
+*QGroundControl* can talk to XBee wireless devices using their proprietary protocol directly on Windows and Linux platforms. This support is not necessary if you're not using XBee devices or aren't using their proprietary protocol. On Windows, the necessary dependencies are included in this repository and no additional steps are required. For Linux, change to the `libs/thirdParty/libxbee` folder and run `make;sudo make install` to install libxbee on your system (uninstalling can be done with a `sudo make uninstall`). *qmake* will automatically detect the library on Linux, so no other work is necessary.
+
+To disable XBee support you may add `DEFINES+=DISABLE_XBEE` to *qmake*.
+
+### Video Streaming
+
+Check the [Video Streaming](https://github.com/mavlink/qgroundcontrol/tree/master/src/VideoStreaming) directory for further instructions.
```

Comment:
@DonLakeFlyer I also propose to pull this into the guide. OK?

Reply:
Or is there a good reason to leave it in the git tree?

Reply:
This one is tricky. Because at the end fo the video streaming line are helpful details for people trying to get video streaming to work. Which is more user guide than dev guide.

Reply:
You mean "video streaming **link**"?

Would we perhaps be better then putting this information in the User Guide and just linking from the Developer Guide?

Reply:
Yeah maybe. It kind of in them middle of user and dev.